VoltAir/qml/actors/Orb.qml

```diff
@@ -34,19 +34,22 @@ Actor {
 
     z: 4
 
-    AnimatedImageRenderer {
+    ImageRenderer {
+        function chooseLetter() {
+            return Math.random() > 0.5 ? "A" : "B";
+        }
         id: graphic
 
         sizeScale: 1.25
-        rotation: Math.random() * 360.0
-        useGameTime: false
+        // rotation: Math.random() * 360.0
```

Comment:
@yorugac We don't want the letters to be rotating. Does that mean that we will not be able to create a rotating effect?